### PR TITLE
WEBUI-647: increase number of runner replicas

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

File: charts/runner/values.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 autoscaling:
-  minReplicas: 1
-  maxReplicas: 4
+  minReplicas: 2
```
Comment (Gabez0r) [inline charts/runner/values.yaml:2]: If we have two hanging processes, we can still have the same issue, right?

Comment (richardsd) [reply charts/runner/values.yaml:2]: Yes. The goal is just to mitigate the problem for now 😬

Comment (Gabez0r) [reply charts/runner/values.yaml:2]: But we only have 16CPU available, right? if so, and if we are bumping resources up to 6 CPU, that means that we probably won't be able to have more than 2 replicas up, right?

Comment (richardsd) [reply charts/runner/values.yaml:2]: 🤔  Yes, we have 16CPU per node (we only have one node, but can scale to 6). Either way, you're right. I would say to bump the `minReplicas` to 2, and maybe keep the CPU, but increase the memory, wdyt?

Comment (Gabez0r) [reply charts/runner/values.yaml:2]: I'm ok with giving 6 a try, and then try to go back to 4 just so we can at least scale up to 4 replicas.

**Review by Gabez0r — APPROVED** [review]

File: charts/runner/values.yaml (hunk 1 above)
Comment (richardsd) [reply charts/runner/values.yaml:2]: Sure, fine by me too 👍

**Pull request merged**